.yarnrc.yml

```diff
@@ -0,0 +1,9 @@
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+    spec: "@yarnpkg/plugin-workspace-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
```

Comment:
The interactive-tools plugin lets you run `yarn upgrade-interactive` to open an interactive CLI for upgrading packages. It'll show you which packages have updates available, and lets you select which ones you want to update. I use it all the time in the other repos (axe-storybook-testing, frontend-libs, along).